core/com.b2international.snowowl.core.rest.tests/.launch/core-e2e-tests.launch

```diff
@@ -195,8 +195,8 @@
         <setEntry value="org.eclipse.xtext.xbase.lib@default:default"/>
         <setEntry value="org.eclipse.xtext@default:default"/>
         <setEntry value="org.glassfish.expressly@default:default"/>
-        <setEntry value="org.hamcrest.core@default:default"/>
-        <setEntry value="org.hamcrest@default:default"/>
+        <setEntry value="org.hamcrest*2.2.0@default:default"/>
+        <setEntry value="org.hamcrest*3.0.0@default:default"/>
```

Comment:
Are both hamcrests required?

Reply:
Hmm, for some reason I thought both is needed, but after a second look it appears to be not the case. Removing the 3.0.0 version.